tutorial/quickstart.md

```diff
@@ -0,0 +1,459 @@
+        for j in range(n):
+            c[i, j] = 0.0
+            for k in range(p):
+                c[i, j] += a[i, k] * b[k, j]
+#$ omp end parallel
+
+    return 0
+```
+We now translate this file to Fortran, and compile it to a Python C extension module, using the command
+```bash
+$ pyccel mod.py --language fortran
+```
+The flag `--language fortran` may be omitted, as Pyccel uses Fortran as the default backend language.
+These are the contents of the current directory:
+```bash
+$ tree .
+.
+├── mod.cpython-36m-x86_64-linux-gnu.so
+├── mod.py
+└── __pyccel__
+    ├── bind_c_mod.f90
+    ├── bind_c_mod.o
+    ├── build
+    │   ├── lib.linux-x86_64-3.6
+    │   └── temp.linux-x86_64-3.6
+    │       └── mod_wrapper.o
+    ├── mod.f90
+    ├── mod.mod
+    ├── mod.o
+    ├── mod_wrapper.c
+    └── setup_mod.py
+
+4 directories, 10 files
+```
+
+And this is the Fortran file `mod.f90`:
+```fortran
+module mod
+
+use, intrinsic :: ISO_C_BINDING
+
+implicit none
+
+contains
+
+!........................................
+function matmul(a, b, c) result(Out_0001)
+
+  implicit none
+
+  integer(C_INT64_T) :: Out_0001
+  real(C_DOUBLE), intent(in) :: a(0:,0:)
+  real(C_DOUBLE), intent(in) :: b(0:,0:)
+  real(C_DOUBLE), intent(inout) :: c(0:,0:)
+  integer(C_INT64_T) :: m
+  integer(C_INT64_T) :: p
+  integer(C_INT64_T) :: q
+  integer(C_INT64_T) :: n
+  integer(C_INT64_T) :: r
+  integer(C_INT64_T) :: s
+  integer(C_INT64_T) :: i
+  integer(C_INT64_T) :: j
+  integer(C_INT64_T) :: k
+
+  m = size(a, 2, C_INT64_T)
+  p = size(a, 1, C_INT64_T)
+  q = size(b, 1, C_INT64_T)
+  n = size(b, 2, C_INT64_T)
+  r = size(c, 2, C_INT64_T)
+  s = size(c, 1, C_INT64_T)
+  if (p /= q .or. m /= r .or. n /= s) then
+    Out_0001 = -1_C_INT64_T
+    return
+  end if
+  !$omp parallel
+  !$omp do schedule(runtime)
+  do i = 0_C_INT64_T, m-1_C_INT64_T, 1_C_INT64_T
+    do j = 0_C_INT64_T, n-1_C_INT64_T, 1_C_INT64_T
+      c(j, i) = 0.0_C_DOUBLE
+      do k = 0_C_INT64_T, p-1_C_INT64_T, 1_C_INT64_T
+        c(j, i) = c(j, i) + a(k, i) * b(k, j)
+      end do
+    end do
+  end do
+  !$omp end parallel
+  Out_0001 = 0_C_INT64_T
+  return
+
+end function matmul
+!........................................
+
+end module mod
+```
+
+### Interactive Usage with `epyccel`
+
+In addition to the `pyccel` command, the Pyccel library provides the `epyccel` Python function, whose name stands for "embedded Pyccel": given a pure Python function `f` with type annotations, `epyccel` returns a "pyccelized" function `f_fast` that can be used in the same Python session.
+For example:
+```python
+from pyccel.epyccel import epyccel
+from mod import f
+
+f_fast = epyccel(f)
+```
+In practice `epyccel` copies the contents of `f` into a temporary python file in the `__epyccel__` directory, then it calls the `pyccel` command to generate a Python C extension module that contains a single pyccelized function, and finally it imports this function and returns it to the caller.
```

Comment:
I think it could be useful to add a sentence about imports here:
```
In practice `epyccel` copies the contents of `f` into a temporary python file in the `__epyccel__` directory. As a result it is important that all imports are written inside the function when using `epyccel`. Once the file has been copied, `epyccel` calls the `pyccel` command to generate a Python C extension module that contains a single pyccelized function. Then finally, it imports this function and returns it to the caller.
```

Reply:
I agree, this is important to mention! Thanks

Reply:
I made the changes you suggested in commit 9131b26